src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -487,9 +491,10 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
     return res.data;
   }
 
-  public async updateLayerFromServiceIfNeeded(reqConfig: RequestConfiguration): Promise<void> {
+  public async updateLayerFromServiceIfNeeded(reqConfig?: RequestConfiguration): Promise<void> {
```

Comment:
This is probably a typo? `fetchLayerParamsFromSHServiceV3` function signature also requests `reqConfig` to be defined.

Reply:
Well because it's a public method I thought it would be better to have the request configuration as optional. For all protected methods this is mandatory but since this one is public I thought it may be a better option.

Reply:
I see... In that case all of them should be optional, until we make a check if the `reqConfig` was supplied or not. But this is probably not within the scope of this MR, so we can leave it as it is... resolving.